source/stylesheets/modules/_breadcrumbs.scss

```diff
@@ -20,13 +16,13 @@
     display: inline-block;
 
     margin: 0;
-    padding: 0 $gutter-one-quarter 0 (11px + $gutter-one-quarter);    
-    
+    padding: 0 $gutter-one-quarter 0 (11px + $gutter-one-quarter);
+
```

Comment:
This is my editor automatically stripping whitespace off the end of lines – decided to keep it in as it does neaten things up a bit :man_shrugging:.